angular/projects/spark-angular/src/lib/components/sprk-promo/sprk-promo.stories.ts

```diff
@@ -114,22 +175,49 @@ withImage.story = {
 export const withReversedImage = () => ({
   moduleMetadata: modules,
   template: `
-    <sprk-promo
-      title="Title"
-      subtitle="Subtitle"
-      mediaRev="true"
-      imgAlt="placeholder"
-      imgSrc="https://spark-assets.netlify.app/house.jpg"
-      imgHref="https://sparkdesignsystem.com"
-      cta="button"
-      ctaText="Learn More"
-      buttonHref="#"
-      idString="promo-4"
-      hasBorder="true"
-    >
-      Lorem ipsum dolor. Sit amet pede. Tempus donec et. Suspendisse id
-      inventore integer eum non enim diam habitant. Maecenas nunc per lacus
-      neque egestas. Diam quod curabitur.
+    <sprk-promo idString="rev-image-promo" hasBorder="true" additionalClasses="sprk-o-Stack--split@s">
+      <div sprkStackItem sprkPromoContent class="sprk-o-Stack__item--half@s">
+        <sprk-stack itemSpacing="large">
+          <h3
+            sprkHeading
+            variant="displayFive"
+            sprkPromoHeading
+            sprkStackItem
+          >
+            Title
+          </h3>
+
+          <p sprkPromoSubheading sprkText variant="bodyOne" sprkStackItem>
+            Subtitle
+          </p>
+
+          <p sprkText variant="bodyTwo" sprkStackItem>
+            Lorem ipsum dolor. Sit amet pede. Tempus donec et. Suspendisse id
+            inventore integer eum non enim diam habitant. Maecenas nunc per lacus
+            neque egestas. Diam quod curabitur.
+          </p>
+
+          <div sprkStackItem>
+            <a
+              sprkLink
+              variant="unstyled"
+              href="#nogo"
+              class="sprk-c-Button"
+              analyticsString="rev-image-promo-cta"
+            >
+              Learn More
+            </a>
+          </div>
+        </sprk-stack>
+      </div>
+
+      <a href="#nogo" sprkStackItem class="sprk-o-Stack__item--half@s">
+        <img
+          sprkPromoMedia
+          alt="A house and front yard."
```

Comment:
Same feedback on the alt text here.

Reply:
Olya said to make this Additional insights about this property. I am updating it